magento/magento2
# Fixed issue .ui-state-focus missing from main navigation in 2.4.5 and 2.4.6

### Description (*)

The bug was reproduced because the updated version of jquery-ui menu widget now adds the "ui-state-active" class instead of "ui-state-focus" to the element with focus. When the jquery-ui library was updated (somewhere between version 2.4.3 and 2.4.5) these edits were added but the styles were not changed. 
In this PR, the selectors for the "ui-state-focus" class have been replaced by "ui-state-active" because the "ui-state-focus" class is no longer added to the menu item with focus in the menu functionality of the jquery-ui library widget

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#37300

### Manual testing scenarios (*)

1. Install a vanilla Magento version 2.4.5 - 2.4.6
2. Open developer tools and inspect a submenu item in the main navigation to watch classes
3. Using keyboard tab to the main menu then use arrow keys to navigate to submenu items
4. When focus is on a menu/submenu item a class should be added of "ui-state-focus" ( Also the menu item with focus must be visually marked )
### Questions or comments


### Contribution checklist (*)
